### PR TITLE
feature: add Structure page

### PR DESCRIPTION
**Pull request by LindonAliu**

### CODE REVIEW

**Review by badermarouan — DISMISSED** [review]

**Review by Snowy1803 — DISMISSED** [review]

File: _docs/structs_in_glu.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+Glu allows you to pack structures for memory efficiency using the `@packed` attribute. This feature exists in other programming languages and can be useful for certain applications where memory usage is critical.
+
+```glu
+@packed struct Data {
+    a: Int8,
+    b: Int64,
+}
+```
+
+In this example, the structure Data is packed, meaning its fields are laid out in memory without padding. While this can save memory, it comes with a trade-off: accessing the members of a packed structure can be slower than accessing members of a non-packed structure due to potential misalignment in memory.
```
Comment (yunsecode) [inline _docs/structs_in_glu.md:84]: better to explain number of bytes before and after `packed`?

File: _docs/structs_in_glu.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+Using the @packed attribute allows you to gain memory efficiency at the cost of execution speed. This trade-off should be considered when deciding whether to pack a structure.
```
Comment (yunsecode) [inline _docs/structs_in_glu.md:86]: add backtick to @packed

File: _docs/structs_in_glu.md (hunk 1 above)
Comment (LindonAliu) [reply _docs/structs_in_glu.md]: i've added the explaination

File: _docs/structs_in_glu.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+}
+
+func main() {
+    let person: Person = { "Alice", 25 };
+    std::print(person.name); // prints "Alice"
+    std::print(person.age); // prints 25
+}
+```
+
+## Packing and Unpacking Structures
```
Comment (Snowy1803) [inline _docs/structs_in_glu.md:73]:
```suggestion
## Packed Structures
```
Not an action

File: _docs/structs_in_glu.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+    a: Int8,
+    b: Int64,
+}
+```
+
+Without the @packed attribute, the compiler may add padding between the fields to align them in memory, improving access speed but using more memory. For example, an `Int8` field followed by an `Int64` field might look like this in memory:
+
+```
+| a (1 byte) | padding (7 bytes) | b (8 bytes) |
+```
```
Comment (Snowy1803) [inline _docs/structs_in_glu.md:99]: Use a table by removing the code block?

Comment (yunsecode) [reply _docs/structs_in_glu.md:99]: +1

**Review by yunsecode — APPROVED** [review]

**Review by Snowy1803 — APPROVED** [review]

**Pull request merged**